plotnine/geoms/annotation_alternating_background.py

```diff
@@ -0,0 +1,127 @@
+from matplotlib.collections import PolyCollection
+import pandas as pd
+import numpy as np
+
+from ..utils import to_rgba, SIZE_FACTOR
+from ..doctools import document
+from ..layer import layer
+from ..coords import coord_flip
+from ..scales.scale import scale_discrete
+from .geom import geom
+from .annotate import annotate
+
+
+@document
+class annotation_alternating_background(annotate):
+    """
+    Alternating backgrounds, centered around each label on x-axis.
+    (Use coord_flip for y-axis)
+
+    Useful for geom_jitter.
+
+    {usage}
+    plot += annotation_alternating_background(fill=['red', 'blue'], alpha=0.3)
+
+    Parameters
+    ----------
+    fills - list of colors to alternate through.
+    Default: ["#AAAAAA", "#CCCCCC"]
+
+    {common_parameters}
+    """
+
+    def __init__(self, **kwargs):
+        self._annotation_geom = _geom_annotation_alternating_background(
+            **kwargs)
+
+    def __radd__(self, gg, inplace=False):
```

Comment:
There is no need for this __radd__.

Being extra helpful to the user by forcing the this annotation to be the lowest layer has two consequences.

1. It limits how creative the user can be with the layer.
2. The code in other places has to be broken-up and rearranged to suit this functionality.

The burden should be on the user to put the layer created by this annotation in the right position. Users of this library learn quickly that they have to manage the order of the layers.